service/src/main/kotlin/fi/espoo/evaka/invoicing/service/FeeDecisionService.kt

```diff
@@ -86,7 +88,11 @@ class FeeDecisionService(
 
         val validIds = validDecisions.map { it.id }
 
-        approveFeeDecisionDraftsForSending(tx.handle, validIds, user.id)
+        val (retroactiveIds, otherValidIds) = validDecisions.partition { isRetroactive(it.validFrom, LocalDate.now()) }
```

Comment:
I guess it would have been polite to do this `LocalDate.now()` refactoring in this PR.